keyboards/handwired/neodox/info.json

```diff
@@ -0,0 +1,133 @@
+{
+    "manufacturer": "MatteoDM",
+    "keyboard_name": "Neodox",
+    "maintainer": "MatteoDM",
+    "backlight": {
+        "driver": "pwm",
+        "levels": 10,
+        "pin": "GP28"
+    },
+    "bootloader": "rp2040",
+    "build": {
+        "debounce_type": "asym_eager_defer_pk"
+    },
+    "diode_direction": "COL2ROW",
+    "features": {
+        "audio": false,
+        "backlight": true,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": false,
+        "quantum_painter": true,
+        "rgblight": true,
+        "wpm": true
+    },
+    "matrix_pins": {
+        "cols": ["GP9", "GP10", "GP11", "GP12", "GP13", "GP14", "GP15"],
+        "rows": ["GP4", "GP5", "GP6", "GP7", "GP8"]
+    },
+    "processor": "RP2040",
+    "rgblight": {
+        "led_count": 15
+    },
+    "split": {
+        "enabled": true,
+        "main": "matrix_grid",
+        "matrix_grid": ["GP5", "GP15"],
+        "transport": {
+            "protocol": "serial",
+            "sync_matrix_state": false,
+            "watchdog": false
+        }
+    },
+    "tapping": {
+        "term": 200,
+        "term_per_key": false
```

Comment:
tapping term per key disabled in `info.json` but enabled in `config.h` - which is to be?

Reply:
you are right thanks , fixed here abb3e6d45804bbf86788c4010cc80b2c31a4f755